core/src/main/java/se2/ticktackbumm/core/screens/WinnerScreen.java

```diff
@@ -99,6 +114,31 @@ public void clicked(InputEvent event, float x, float y) {
         sprite.setRegionHeight(TickTackBummGame.HEIGHT);
     }
 
+
+    public void initTables(Table[] winnerTables) {
+        Player player = new Player();
+        player = placedPlayers [0];
+        placedPlayers[0] = placedPlayers[1];
+        placedPlayers[1] = player;
```

Comment:
Du brauchst das hier nicht tauschen. Entferne einfach das `.reversed()` in `ServerData`, dann sind die in der richtigen Reihenfolge.